lib/new_relixir.ex

```diff
@@ -40,4 +40,11 @@ defmodule NewRelixir do
   def configured? do
     Application.get_env(:new_relixir, :application_name) != nil && Application.get_env(:new_relixir, :license_key) != nil
```

Comment:
Shouldn't this be change too?

Reply:
@peburrows ping

Reply:
Sorry, I started using Appsignal soon after submitting this PR and hadn't looked at this library since then, but yes, I believe you are correct that this line should change, too.